pandas/tests/reshape/test_reshape.py

```diff
@@ -217,34 +217,36 @@ def test_multiindex(self):
 
 
 class TestGetDummies(object):
```

Comment:
instead of doing this define a fixture that returns the various dtypes that you are testing